dev/CommonStyles/CommandBar_themeresources.xaml

```diff
@@ -830,6 +829,7 @@
                                 <!-- Use a ContentControl rather than a ContentPresenter so that IsEnabled can be set to false
                                      in the Minimal/HiddenClosed states to remove it from being a tab-stop candidate. -->
                                 <ContentControl x:Name="ContentControl"
+                                      Margin="{TemplateBinding Padding}"
```

Comment:
Should we backport those changes to the V1 style?

Also, great PR description, definitely made me chuckle 😄

Reply:
good question

Reply:
@teaP I don't understand how did TemplateBinding to 4,0,0,0 margin make such a change in the visuals?

Reply:
I suspect that is not ensuring the same settings for the before and after screenshots.  The label position is a setting you can change.